src/components/Places.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useMemo } from "react";
+import InfiniteScroll from "react-infinite-scroll-component";
+import { useInfiniteQuery } from "react-query";
+import { getPlaces, ResponsePlace } from "../services/CharacterService";
+import PlaceCard from "./PlaceCard";
+import Spinner from "./Spinner";
+
+const Places = () => {
+  useEffect(() => {
+    var edgeSize = 150;
+    var timer = setTimeout(function () {}, 0);
+
+    function handleMousemove(event: MouseEvent) {
```

Comment:
Imma gonna go ahead and assume this entire useEffect was copy & pasted from somewhere else because of the use of var
You'll have a fun time once you add eslint to this repo :)